### ISSUE TITLE
Allow TAs/Tutors/AIs to see a list of students they've helped in past OH

### ISSUE THREAD

**Issue by vivrekar** [Cal-CS-61A-Staff/oh-queue#141]
Helpful in case we want to follow-up with a student or clarify a statement we made while helping them, in case OH ends before we can finish, or we say something misleading while helping them.

**Comment by chrononyan** [comment]
Can this be merged with #129?

**Comment by vivrekar** [comment]
Yes let's do that!

**Comment by ja5087** [comment]
Duplicate of #129
